### PR TITLE
python311Packages.jaxtyping: 0.2.28 -> 0.2.31

### PR DESCRIPTION
**Pull request by GaetanLepage**
## Description of changes

Changelog:
https://github.com/patrick-kidger/jaxtyping/releases/tag/v0.2.31

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
  </ul>
</details>
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
  </ul>
</details>
<details>
  <summary>26 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages failed to build:</summary>
  <ul>
    <li>python311Packages.equinox</li>
    <li>python311Packages.equinox.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
    <li>python312Packages.equinox</li>
    <li>python312Packages.equinox.dist</li>
  </ul>
</details>
<details>
  <summary>28 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages failed to build:</summary>
  <ul>
    <li>python311Packages.equinox</li>
    <li>python311Packages.equinox.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
    <li>python312Packages.equinox</li>
    <li>python312Packages.equinox.dist</li>
  </ul>
</details>
<details>
  <summary>28 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>36 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.equinox</li>
    <li>python311Packages.equinox.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.equinox</li>
    <li>python312Packages.equinox.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>32 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>28 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>36 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.equinox</li>
    <li>python311Packages.equinox.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.equinox</li>
    <li>python312Packages.equinox.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>32 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>28 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>36 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.equinox</li>
    <li>python311Packages.equinox.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.equinox</li>
    <li>python312Packages.equinox.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 315102` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>36 packages built:</summary>
  <ul>
    <li>python311Packages.ax</li>
    <li>python311Packages.ax.dist</li>
    <li>python311Packages.botorch</li>
    <li>python311Packages.botorch.dist</li>
    <li>python311Packages.equinox</li>
    <li>python311Packages.equinox.dist</li>
    <li>python311Packages.gpytorch</li>
    <li>python311Packages.gpytorch.dist</li>
    <li>python311Packages.jaxtyping</li>
    <li>python311Packages.jaxtyping.dist</li>
    <li>python311Packages.linear-operator</li>
    <li>python311Packages.linear-operator.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pyannote-pipeline</li>
    <li>python311Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.ax</li>
    <li>python312Packages.ax.dist</li>
    <li>python312Packages.botorch</li>
    <li>python312Packages.botorch.dist</li>
    <li>python312Packages.equinox</li>
    <li>python312Packages.equinox.dist</li>
    <li>python312Packages.gpytorch</li>
    <li>python312Packages.gpytorch.dist</li>
    <li>python312Packages.jaxtyping</li>
    <li>python312Packages.jaxtyping.dist</li>
    <li>python312Packages.linear-operator</li>
    <li>python312Packages.linear-operator.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/jaxtyping/default.nix [hunk 1]
```diff
@@ -30,15 +30,15 @@ let
...
+    build-system = [
       hatchling
       pythonRelaxDepsHook
     ];
 
-    propagatedBuildInputs = [
+    dependencies = [
       numpy
       typeguard
       typing-extensions
```
Comment (natsukium) [inline pkgs/development/python-modules/jaxtyping/default.nix:44]:
```suggestion
      typeguard
```
https://github.com/patrick-kidger/jaxtyping/compare/v0.2.28...v0.2.31#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R26

Comment (GaetanLepage) [reply pkgs/development/python-modules/jaxtyping/default.nix]: Indeed I missed that...

**Review by natsukium — APPROVED** [review]
Sorry for the late review. LGTM.

**Comment by GaetanLepage** [comment]
> Sorry for the late review. LGTM.

No worry, thank you very much !

**Pull request merged**